content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -1,13 +1,17 @@
 ---
+reviewers:
+- sig-cluster-lifecycle
```

Comment:
This is unnecessary. The SIG members won't have bandwidth to review all localizations
We should instead remove the `reviewers` when translating a page. Otherwise, the change will be notified to the SIG and they will get confused.

Reply:
understood, I was thinking this is only a note, thx